apps/crc_interactive.py

```diff
@@ -51,7 +51,7 @@ def __init__(self) -> None:
         resource_args = self.add_argument_group('Arguments for Increased Resources')
         resource_args.add_argument('-b', '--mem', type=int, default=self.default_mem, help='memory in GB')
         resource_args.add_argument(
-            '-t', '--time', type=int, default=self.default_time,
+            '-t', '--time',  default=self.default_time,
             help=f'run time in hours [default: {self.default_time}]')
```

Comment:
It may be worth updating the help text to reflect new functionality.

Reply:
Users will still get the exact time they requested since their minute argument is being passed on 1-1 to srun. It is just in the validity check for  maximal time constraints where the argument is being rounded up/down.